static/js/StaticPages.jsx

```diff
@@ -3126,6 +3129,330 @@ const JobsPage = memo(() => {
     );
 });
 
+
+/*
+* Products Page
+*/
+
+
+
+const ProductsPage = memo(() => {
```

Comment:
Okay! This was left in to copy the patterns we have on the job and team pages, which are also static / powered by Strapi. Should we refactor everything?

Reply:
I think I added the React memoization to the other components to prevent extraneous impressions for analytics when using the OnInView component I made. It's also better, because if the props stay constant, the component won't re-render when the viewport is resized